docs/en/tech_notes.md

```diff
@@ -21,7 +17,6 @@ are in place)
 and SAML components.
 
 * `LDAPAuthenticator::authenticate`: creates stub `Member` after authorisation (if non-existent).
-* `LDAPMemberExtension::memberLoggedIn`: triggers LDAP synchronisation, rewriting all `Member` fields.
```

Comment:
Keep this and change to `LDAPMemberExtension::afterMemberLoggedIn()`

Reply:
DONE